srcs/runners_manager/vm_creation/gcloud/schema.py

```diff
@@ -11,3 +11,4 @@ class GcloudConfigVmType(Schema):
     project = fields.Str(required=True)
     family = fields.Str(required=True)
     disk_size_gb = fields.Str(required=True)
+    spot = fields.Boolean(missing=False)
```

Comment:
I have reverted back to using `spot` as it makes more sense to me. Especially since we are changing the variable for both preemptible and spot instances.